staging/src/k8s.io/component-base/logs/logs.go

```diff
@@ -41,13 +41,32 @@ const deprecated = "will be removed in a future release, see https://github.com/
 
 var (
 	packageFlags = flag.NewFlagSet("logging", flag.ContinueOnError)
-	logFlushFreq time.Duration
 	logrFlush    func()
+
+	// Periodic flushing gets configured either via the global flag
+	// in this file or via LoggingConfiguration.
+	logFlushFreq        *time.Duration
+	packageLogFlushFreq time.Duration
```

Comment:
Looking into code logFlushFreq is only used to store information whether flag was initialized via BindLoggingFlags or via  AddFlags. Let me know if I misunderstood, but could we rather store this information in bool rather than pointer?

Reply:
We can. I'll change it.

My initial thinking was that I would point to the actual value (from this file or options.go) and use the same code, but that would have had problems when Options.Apply gets called after InitLogs.

Reply:
Done.